### PR TITLE
Add Ed25519Signature2020 cryptosuite.

### PR DESCRIPTION
**Pull request by msporny**
This PR adds the Ed25519Signature2020 cryptosuite, as requested in issue #10.

### CODE REVIEW

**Comment by jandrieu** [comment]
Hmmm...

I have a feeling the code owner's file is out of date. I didn't realize I was an editor of this registry...

**Comment by msporny** [comment]
> I have a feeling the code owner's file is out of date. I didn't realize I was an editor of this registry...

Welcome to the long forgotten baby of the VCWG... :P

**Review by peacekeeper — APPROVED** [review]
Good!

**Review by mprorock — APPROVED** [review]

File: index.html
Comment (clehner) [inline index.html:201]: Does this mean this is a reference implementation?

Should other registered proof/extension types also link to one or more implementations?

Comment (msporny) [reply index.html:201]: It's meant to convey that it is *an* implementation, not necessarily *the* implementation. We try to stay away from saying "reference" implementation because sometimes multiple people think they are creating the reference implementation. :)

Is there a better way this could be conveyed?

Comment (clehner) [reply index.html:201]: That makes sense, thanks.

I think an implementations column could work, or having multiple implementations listed in some way, that could make it more clear.

I'm okay with letting future PRs figure it out (how to list more than one implementation); this PR already has approvals as-is.

Comment (msporny) [reply index.html:201]: Ok, then I'll merge this one and raise another one to add that column.

Comment (msporny) [reply index.html:201]: Raised PR https://github.com/w3c-ccg/vc-extension-registry/pull/12 to address @clehner's comment above. It turned into a larger refactor, but one that I hope the community will accept.

**Comment by charlestonxorg** [comment]
Oof

**Comment by TallTed** [comment]
I cannot report [this spam comment/user](https://github.com/w3c-ccg/vc-extension-registry/pull/11#issuecomment-1374712292), but hopefully a CODEOWNER or similar can?

**Pull request merged**
